manager/assets/modext/core/modx.js

```diff
@@ -496,6 +496,37 @@ Ext.extend(MODx,Ext.Component,{
     ,isEmpty: function(v) {
         return Ext.isEmpty(v) || v === false || v === 'false' || v === 'FALSE' || v === '0' || v === 0;
     }
+    
+    ,createResource: function(record) { 
+        if (MODx.createResourceWindow) {
+            MODx.createResourceWindow.destroy();
+        }
+        
+        MODx.createResourceWindow = MODx.load({
+            xtype       : 'modx-window-create-resource',
+            record      : record,
+            closeAction : 'close',
+            listeners   : {
+                'success'   : {
+                    fn          : function(r) {
+                        MODx.loadPage('?a=resource/update&id=' + r.a.result.object.id);
+                    },
+                    scope       : this
+                },
+                'failure'   : {
+                    fn          : function(data, data2) {
+                        console.log('failure');
+                        console.log(data);
+                        console.log(data2);
```

Comment:
These `console.log` statements should be replaced with "proper" error handling. That could be as simple as an alert showing the failure reason or a `MODx.Msg`